docs/source/upcoming_release_notes/1160-Update_lxt_limits.rst

```diff
@@ -0,0 +1,30 @@
+1160 Update lxt limits
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- Updates limits for LaserTiming, LaserTimingCompensation, and LxtTtcExample from +/-10us to +/-100us
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- N/A
+
+Maintenance
+-----------
+- Update limits from +/-10us to +/-100us
+
+Contributors
+------------
+- espov
```

Comment:
```suggestion
- vespos
```

Reply:
Do we use github or slac username here?

Reply:
I'm going to force accept this suggestion, approve, and merge after the ci re-runs